common/socket/src/main/java/io/helidon/common/socket/PlainSocket.java

```diff
@@ -80,7 +88,7 @@ public static PlainSocket server(Socket delegate, String channelId, String serve
      * @return a new plain socket
      */
     public static PlainSocket client(Socket delegate, String channelId) {
-        return new PlainSocket(delegate, channelId, "client");
+        return new PlainSocket(delegate, channelId, "client", null);
```

Comment:
We cannot use `null` in API

Reply:
Obsolete. The PR no longer affects `PlainSocket`.